.yarnrc.yml

```diff
@@ -26,3 +26,36 @@ plugins:
 
 yarnPath: .yarn/releases/yarn-3.2.3.cjs
 enableScripts: true
+
+packageExtensions:
+  "@tanstack/query-persist-client-core@4.22.0":
+    dependencies:
+      "@tanstack/query-core": "4.22.0"
+  "hardhat-watcher@^2.1.1":
+    dependencies:
+      "hardhat": "^2.0.0"
+  "@uniswap/widgets@^2.16.2":
+    dependencies:
+      "@walletconnect/ethereum-provider": "1.7.8"
+  "@uniswap/redux-multicall@^1.1.1":
+    dependencies:
+      "@ethersproject/abi": "5"
+      "@ethersproject/bignumber": "5.7.0"
+      "@ethersproject/contracts": "5.7.0"
+  "styled-components@>=5":
+    dependencies:
+      "react-is": "16.8.0"
+  "abitype@^0.1.8":
+    dependencies:
+      "typescript": "^4.7.4"
+  "eth-block-tracker@4.4.3":
+    dependencies:
+      "@babel/core": "7.4.0"
+
+logFilters:
+  - pattern: "@ubiquity/dapp@workspace:packages/dapp provides react * with version * which doesn't satisfy what @uniswap/widgets and some of its descendants request"
```

Comment:
Will this filter only for Uniswap package dependents related logs then? Otherwise seems like a potentially bad workaround that can hide relevant errors down the road.

Reply:
It'll only resolve the `@uniswap/widgets`'s issue as it uses 

> react: ">=17.0.1"
(which is compatible with any react version equal to or above `17.0.1` (`...18.2.0...`) according to semver)

the following package also duplicates the same dependencies into `deps` and `peer-deps` (which is the root of this issue)

@pavlovcik